modules/RTC/TraceablePeerConnection.js

```diff
@@ -1883,6 +1885,50 @@ TraceablePeerConnection.prototype._insertUnifiedPlanSimulcastReceive
         });
     };
 
+/**
+ * Sets the max bitrate on the RTCRtpSender so that the
+ * bitrate of the enocder doesn't exceed the configured value.
+ * This is needed for the desktop share until spec-complaint
+ * simulcast is implemented.
+ * @param {JitsiLocalTrack} localTrack - the local track whose
+ * max bitrate is to be configured.
+ */
+TraceablePeerConnection.prototype.setMaxBitRate = function(localTrack) {
+    const mediaType = localTrack.type;
+
+    if (mediaType !== MediaType.AUDIO) {
```

Comment:
nit: can we do:
```
if (mediaType === MediaType.AUDIO) {
  return;
}
```
to avoid the extra layer of indentation for everything below it?

Reply:
sure